com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryActionEditor.cs

```diff
@@ -108,6 +108,60 @@ public class VRCFuryActionDrawer : PropertyDrawer {
 
                 return row;
             }
+            case nameof(PoiyomiUVTileAction): {
+                var content = new VisualElement();
+                var row = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var label = new Label("Poiyomi UV Tile Discard") {
+                    style = {
+                        flexGrow = 0,
+                        flexBasis = 140,
+                    }
+                };
+                row.Add(label);
+
+                var propField = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("renderer"));
+                propField.style.flexGrow = 1;
+                row.Add(propField);
+            
+                var propField2 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot1"), tooltip: "UV X Axis, Val 0-3");
+                propField2.style.flexGrow = 0;
+                propField2.style.flexBasis = 30;
+                row.Add(propField2);
+
+                var propField3 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot2"),tooltip: "UV Y Axis, Val 0-3");
+                propField3.style.flexGrow = 0;
+                propField3.style.flexBasis = 30;
+                row.Add(propField3);
+                content.Add(row);
+
+                var row2 = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var adv = new Foldout {
+                    text = "Advance Options",
+                    style = {
+                        flexDirection = FlexDirection.Column,
+                    },
+                    value = false
+                };
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("invert"), "Invert the UV Tile Discarding."));
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("toggled"), "Only Add Toggled On clip."));

```

Comment:
Sorry for the delay, I don't get as much time to work on these pulls as I'd like 😢

Is there a purpose for `toggled`? I can't think of why someone would want the resting state to be on if the action they are creating here also turns it on.

Reply:
I didn't find a convenient name for the variable, but the example is. I have a piece of clothing, for instance, shorts, that I want to have when toggled on to turn on the UV tile of the underwear. I don't want it to have it forced off UV tile when the clothing itself is off.